.ci/packaging.groovy

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env groovy
 
-@Library('apm@current') _
+@Library('apm@test-build-error') _
```

Comment:
Will be eventually reverted

Reply:
https://apm-ci.elastic.co/job/apm-shared/job/apm-pipeline-library-mbp/job/master/1057/ is on-going as soon as it's done `current` tag will be updated with the latest changes :)

```suggestion
@Library('apm@current') _
```